src/tests/tablet/schedule/CreateMeetingIfNotHasBody.java

```diff
@@ -0,0 +1,70 @@
+package tests.tablet.schedule;
+
+import static framework.common.AppConfigConstants.EXCEL_INPUT_DATA;
+
+import java.io.IOException;
+import java.net.MalformedURLException;
+import java.util.List;
+import java.util.Map;
+
+import org.junit.Assert;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.Test;
+
+import framework.pages.tablet.HomeTabletPage;
+import framework.pages.tablet.SchedulePage;
+import framework.rest.RootRestMethods;
+import framework.utils.readers.ExcelReader;
+
+/**
+ * TC18: Verify that a meeting can be created if the body is empty
+ * TC28: Verify that pressing enter confirms the name of an attendee
+ * TC29: Verify that Pressing semicolon confirms the name of an attendee
+ * @author Asael Calizaya
+ *
+ */
+public class CreateMeetingIfNotHasBody {
+	private ExcelReader excelReader = new ExcelReader(EXCEL_INPUT_DATA);
+	private List<Map<String, String>> meetingData = excelReader.getMapValues("MeetingData");
+
+	private String subject = meetingData.get(0).get("Subject");
+	private String organizer = meetingData.get(0).get("Organizer");	
+	private String password = meetingData.get(0).get("Password");
+	private String roomName = meetingData.get(1).get("Room");
+	private String authentication = organizer + ":" + password;
+
+	@Test(groups = {"FUNCTIONAL", "UI"})
+	public void testCreateMeetingIfNotHasBody() {
+		String startTime = meetingData.get(0).get("Start time");
+		String endTime = meetingData.get(0).get("End time");
+		String attendee1 = meetingData.get(0).get("Attendee");
+		String attendee2 = meetingData.get(0).get("Attendee");
+		HomeTabletPage homePage = new HomeTabletPage();
+		SchedulePage schedulePage = homePage.clickScheduleBtn();
+		schedulePage
+		.setOrganizerTxtBox(organizer)
+		.setSubjectTxtBox(subject)
+		.setStartTimeDate(startTime)
+		.setEndTimeDate(endTime);
+
+		//Assertion TC28
```

Comment:
instead of do the assertion of all TCs here you should created another test
@Test.....it do more clearly what you are testing